pkg/arvo/sys/zuse.hoon

```diff
@@ -6512,8 +6512,13 @@
       ?^  t.rax
         [p.pok [ire q.pok]]:[pok=$(rax t.rax) ire=i.rax]
       =/  raf/(like term)
-          =>  |=(a/@ ((sand %tas) (crip (flop (trip a)))))
-          (;~(sfix (sear . sym) dot) [1^1 (flop (trip i.rax))])
+        %-  ;~  sfix
+              %+  sear
+                |=(a/@ ((sand %tas) (crip (flop (trip a)))))
+              (cook |=(a/tape (rap 3 ^-((list @) a))) (star aln))
```

Comment:
Also, to make sure I'm clear on this: `sym` -> `(star aln)` is your only real change, right? And that affects the `.woff2` case because it's parsed as `2ffow.`, correct? And then `.7z` example from above still won't pass, because it's sanity-checked as `7z`, which isn't a valid `@tas`.

Reply:
It's parsed as `2ffow` without the `dot`, but yeah you get the gist.

I was trying to find the correct parser here, `urs:ab` and similar don't work in this context since they include `dot` and we're inside `;~  sfix  ...  dot` here. Not sure if `aln` is the correct one tho.

Reply:
Ohh, right. That's a bit awkward, yeah. I'm struggling to find an RFC that says anything about file extensions. Seems like they might just be treated as part of the path, following the same parsing rules? Though, I don't think I've ever seen an extension fancier than `aln` in the wild...